test/test_image.py

```diff
@@ -48,3 +49,36 @@ def test_gradcheck(self):
         input = utils.tensor_to_gradcheck_var(input)  # to var
         assert gradcheck(image.gaussian_blur, (input, kernel_size, sigma,),
                          raise_exception=True)
+
+
+@pytest.mark.parametrize("window_size", [5])
+def test_get_laplacian_kernel(window_size):
+    kernel = image.get_laplacian_kernel(window_size)
+    assert kernel.shape == (window_size,)
+    assert kernel.sum().item() == pytest.approx(0.0)
+
+
+@pytest.mark.parametrize("window_size", [7])
+def test_get_laplacian_kernel2d(window_size):
+    kernel = image.get_laplacian_kernel2d(window_size)
+    assert kernel.shape == (window_size, window_size)
+    assert kernel.sum().item() == pytest.approx(0.0)
+    expected = torch.tensor([[1., 1., 1., 1., 1., 1., 1.],
+                            [1., 1., 1., 1., 1., 1., 1.],
+                            [1., 1., 1., 1., 1., 1., 1.],
+                            [1., 1., 1., -48., 1., 1., 1.],
+                            [1., 1., 1., 1., 1., 1., 1.],
+                            [1., 1., 1., 1., 1., 1., 1.],
+                            [1., 1., 1., 1., 1., 1., 1.]])
+    assert_allclose(expected, kernel)
+
+
+class TestLaplacian:
+    @pytest.mark.parametrize("batch_shape",
+                             [(1, 4, 8, 15), (2, 3, 11, 7)])
+    def test_laplacian(self, batch_shape, device_type):
+        kernel_size = 5
+
+        input = torch.rand(batch_shape).to(torch.device(device_type))
+        laplace = image.Laplacian(kernel_size)
+        assert laplace(input).shape == batch_shape
```

Comment:
You might also need a gradient check function. Additionally, try to play with jitted module. The plan is also to support but it's something that I haven't tried at all